### PR TITLE
Some code styling, thanks to nightly fmt and clippy

### PR DESCRIPTION
**Pull request by omid**
Apply some code styling and add them to the Makefile for the future!
Includes also some tiny logical changes. I'll highlight them with comments.
This change is so opinionated, feel free to comment or reject :D

### CODE REVIEW

File: macros/src/actix.rs [hunk 1]
```diff
@@ -809,9 +809,9 @@ fn handle_unnamed_field_struct(fields: &FieldsUnnamed, props_gen: &mut proc_macr
 }
 
 /// Checks for `api_v2_empty` attributes and removes them.
-fn extract_openapi_attrs<'a>(
-    field_attrs: &'a [Attribute],
-) -> impl Iterator<Item = Punctuated<syn::NestedMeta, syn::token::Comma>> + 'a {
+fn extract_openapi_attrs(
+    field_attrs: &'_ [Attribute],
+) -> impl Iterator<Item = Punctuated<syn::NestedMeta, syn::token::Comma>> + '_ {
```
Comment (omid) [inline macros/src/actix.rs:814]: This is a tiny logical change

File: core/src/v2/schema.rs [hunk 2]
```diff
@@ -294,10 +294,11 @@ impl Apiv2Schema for serde_yaml::Value {}
...
     fn raw_schema() -> DefaultSchemaRaw {
-        let mut schema = DefaultSchemaRaw::default();
-        schema.data_type = Some(T::data_type());
-        schema.format = T::format();
-        schema
+        DefaultSchemaRaw {
+            data_type: Some(T::data_type()),
+            format: T::format(),
+            ..Default::default()
+        }
```
Comment (omid) [inline core/src/v2/schema.rs:301]: This is a tiny logical change

File: core/src/v2/schema.rs [hunk 3]
```diff
@@ -373,10 +374,11 @@ macro_rules! impl_schema_array {
...
             fn raw_schema() -> DefaultSchemaRaw {
-                let mut schema = DefaultSchemaRaw::default();
-                schema.data_type = Some(DataType::Array);
-                schema.items = Some(T::schema_with_ref().into());
-                schema
+                DefaultSchemaRaw {
+                    data_type: Some(DataType::Array),
+                    items: Some(T::schema_with_ref().into()),
+                    ..Default::default()
+                }
```
Comment (omid) [inline core/src/v2/schema.rs:381]: This is a tiny logical change

File: core/src/v2/schema.rs [hunk 4]
```diff
@@ -386,10 +388,11 @@ macro_rules! impl_schema_map {
...
             fn raw_schema() -> DefaultSchemaRaw {
-                let mut schema = DefaultSchemaRaw::default();
-                schema.data_type = Some(DataType::Object);
-                schema.extra_props = Some(Either::Right(V::schema_with_ref().into()));
-                schema
+                DefaultSchemaRaw {
+                    data_type: Some(DataType::Object),
+                    extra_props: Some(Either::Right(V::schema_with_ref().into())),
+                    ..Default::default()
+                }
```
Comment (omid) [inline core/src/v2/schema.rs:395]: This is a tiny logical change

**Review by wafflespeanut — APPROVED** [review]
Nice! Thanks!

**Pull request merged**